test/mocha.opts

```diff
@@ -2,5 +2,3 @@
 --reporter dot
 --recursive
 test/utils/setup.js
-test/integration/{,**/}*.spec.js
```

Comment:
I use a jetbrains IDE, and that doesn't seem to be an option. Up to this point I have been deleting the offending code and avoiding committing it.  While I very much dislike duplication, it seems necessary here.